### DISCUSSION TITLE
How can we have prices on the chart instead of next to the chart like the image below

### DISCUSSION THREAD

**Discussion by mirzaeianfar** [tradingview/lightweight-charts#885] [Q&A]
![group69](https://user-images.githubusercontent.com/53333713/141251536-81c531fb-6857-42ba-95a7-45576e19dc3a.jpg)
Maybe this problem can be changed by changing the background price axis color to transparent so how can this problem be solved?

**Comment by timocov** [comment]
Duplicate of #151

**Reply by timocov** [reply]
See https://github.com/tradingview/lightweight-charts/blob/master/docs/baseline-series.md

**Comment by mirzaeianfar** [comment]
Wrong I did not mean baseline, I meant the prices on the left side of the chart which are plotted vertically
Below the prices is a red line
![group69](https://user-images.githubusercontent.com/53333713/141419046-83714e9f-30ec-45a6-9c67-17dccb4633d7.jpg)

**Comment by mirzaeianfar** [comment]
Now in the image below, the prices are next to the chart, if I want them to be on the chart, so that space can be used optimally in mobile size sizes.
![139635783-654c923e-0990-4756-84a1-a030f2e48aaf](https://user-images.githubusercontent.com/53333713/141419661-a945a231-943a-41a3-aa41-9ce6a3592734.png)

**Reply by timocov** [reply]
See #208

**Comment by mirzaeianfar** [comment]
Thank you
How long do you think this feature will take to be added?
